.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file is not needed, preferably delete it

Reply:
Thanks for noticing, Deleted it.